projects/js-packages/connection/components/connection-error-notice/index.jsx

```diff
@@ -21,7 +22,9 @@ const ConnectErrorNotice = () => {
 				<Notice status={ 'error' } isDismissible={ false } className={ styles.notice }>
 					<Icon icon={ warning } />
 					<div className={ styles.message }>{ errors[ 0 ].error_message }</div>
-					<a className={ styles.link }>{ __( 'Restore Connection', 'jetpack' ) }</a>
+					<Button variant="external-link" weight="bold" className={ styles.link }>
```

Comment:
After chatting, we realized the [`external-link` variant is deprecated](https://github.com/Automattic/jetpack/pull/24505). Please use the isExternalLink prop instead.

Reply:
External link shouldn't fit here cause the link is local